src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class WorkWithFile {
+    private static final String[] STATISTICS_FIELDS = {"supply", "buy", "result"};
+    private static final int SUPPLY_POSITION = 0;
+    private static final int BUY_POSITION = 1;
+    private static final int RESULT_POSITION = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileContent = readFromFile(fromFileName);
+        int[] statisticsValues = calculateStatistics(fileContent);
+        writeToFile(statisticsValues, toFileName);
+    }
+
+    private String readFromFile(String fromFileName) {
+        StringBuilder sb = new StringBuilder();
+
+        try (FileReader reader = new FileReader(fromFileName);
+                BufferedReader bufferedReader = new BufferedReader(reader)) {
+            String readLine = bufferedReader.readLine();
+            while (readLine != null) {
+                sb.append(readLine).append(System.lineSeparator());
+                readLine = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            System.out.println("Error while reading a file");
+        }
+
+        return sb.toString().trim();
+    }
+
+    private int[] calculateStatistics(String fileContent) {
+        int[] statisticsValues = new int[3];
+
+        Scanner scanner = new Scanner(fileContent);
+        while (scanner.hasNextLine()) {
+            String[] line = scanner.nextLine().split(",");
+            if (line[0].equals(STATISTICS_FIELDS[SUPPLY_POSITION])) {
+                statisticsValues[SUPPLY_POSITION] += Integer.parseInt(line[1]);
+            } else {
+                statisticsValues[BUY_POSITION] += Integer.parseInt(line[1]);
```

Comment:
These hardcoded 0/1 indexes also will look better when extracted to constants with proper naming.
N.B.
Please do not reuse existing constant for another purpose even if it has same value

Reply:
I am sorry, but creating own constant for every number in code just seems absurd. I see that you mean better code readability by this, but simple comment, in that exact case, would explain everything much better, in my opinion.

Reply:
comments in the code are a very bad practice, if there is a need to leave a comment, then something is wrong with the code, well, or some very tricky construction that cannot be done in any other way, then it is acceptable.
Usually the code itself should be readable. But such numbers, I have no idea what is there at index 1. Therefore, it is great to give names so that it is clear what exactly we will get